api/nakadi-event-bus-api.yaml

```diff
@@ -351,6 +353,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+    delete:
+      tags:
+        - schema-registry-api
+      security:
+        - oauth2: [nakadi.admin]
+      description: |
+        Deletes an `EventType` identified by its name. The underlying Kafka topic and all events of this `EventType`
```

Comment:
I would rather speak of the "underlying" infrastructure instead of explicitly talking about Kafka.